### DISCUSSION TITLE
Ethernet Module

### DISCUSSION THREAD

**Discussion by parabyte** [pschatzmann/arduino-audio-tools#1617] [Q&A] [answered]
what would be the route you would take to enable a ethernet module like the w5500 with its library to operate with your library instead of using the esp's wifi, 

some pointers be really fantastic to help me along on my way,

**Comment by pschatzmann** [comment, marked as answer]
Just initialize your module yourself and pass the client in the constructor of the URLStream.

If it is for a microcontroller that does not support networking out of the box (e.g STM32), you would need to activate the URLStream in the AudioConfig.h as well.
